lms/djangoapps/monitoring/scripts/clean_unmapped_view_modules.py

```diff
@@ -0,0 +1,67 @@
+"""
+This script outputs a unique and clean set of external apps provided a CSV of unmapped view_func_modules.
```

Comment:
Might be good to mention up here that this is drawn from monitoring data.

Reply:
Added additional comments.  Note: I don't think this script will be used again, but who knows.